clang/lib/Driver/Driver.cpp

```diff
@@ -8469,7 +8477,8 @@ class ToolSelector final {
   /// Return true if a preprocessor action can be collapsed.
```

Comment:
nit: This comment can be updated to include the new return condition.

Reply:
Same remark as above